webroot/results/person.php

```diff
@@ -66,6 +66,17 @@ function showBody () {
   if( $picture )
     echo "<center><img class='person' src='$picture' /></center>";
 
+  #--- Show the In Memoriam if any.
+  $inMemoriamArray = array(
+    "2008COUR01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=2028",
+    "2003LARS01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=1982",
+    "2012GALA02" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=1044",
+    "2008LIMR01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=945",
+    "2008KIRC01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=470",
+  );
+  if( array_key_exists($chosenPersonId, $inMemoriamArray) )
```

Comment:
Ah yeah, our php code definitely was not written @jfly style. Generally I'm off the opinion "follow the style of the code around you", but this is one style thing I don't like to budge on because IMO, it's not just cosmetic.